subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -586,41 +600,16 @@ uint8_t ll_cig_remove(uint8_t cig_id)
 
 	/* CIG exists and is not active */
 	handle_iter = UINT16_MAX;
-	has_cis = false;
 
-	for (uint8_t i = 0U; i < cig->lll.num_cis; i++)  {
+	for (uint8_t i = 0U; i < cig->cis_count; i++)  {
```

Comment:
This change has no actual impact, so we can use any of the two. I think I had the change in because it's central code. But we can use `cig->lll.num_cis` as it also makes good sense in terms of it controlling the allocated instances.